metaconfig-json/src/main/scala/metaconfig/JsonSchema.scala

```diff
@@ -0,0 +1,112 @@
+package metaconfig
+
+import metaconfig.generic.Setting
+import metaconfig.generic.Settings
+import ujson._
+
+object JsonSchema {
+
+  def generate[T: ConfEncoder](
+      title: String,
+      description: String,
+      url: Option[String],
+      default: T)(implicit settings: Settings[T]): Js.Obj = {
+    ConfEncoder[T].write(default) match {
+      case obj: Conf.Obj =>
+        this.generate[T](title, description, url, obj)
+      case els =>
+        throw new IllegalArgumentException(s"Expected Conf.Obj, got $els")
+    }
+  }
+
+  def generate[T](
+      title: String,
+      description: String,
+      url: Option[String],
+      default: Conf.Obj)(implicit settings: Settings[T]): Js.Obj = {
+
+    val properties: List[(String, Js.Obj)] = settings.settings
+      .zip(default.values)
+      .map { case (s, (_, v)) => fromSetting(s, v) }
+    pprint.log(properties)
```

Comment:
Did you leave this (and the one below) here intentionally?

Reply:
Good catch! Those were debugging leftovers, removed now